com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/AbstractSelectorFragmentProvider.java

```diff
@@ -118,6 +124,10 @@ public boolean appendFragmentSegment(final EObject object, final StringBuilder b
     return appendFragmentSegmentFallback(object, builder);
   }
 
+  private Boolean isCaseSensitive(final EClass eClass) {
+    return (Boolean) eclassToCaseSensitive.computeIfAbsent(eClass, k -> eClass.getEAllSuperTypes().stream().map(EClass::getName).anyMatch(n -> "ICaseSensitiveNamedElement".equals(n))); //$NON-NLS-1$

```

Comment:
I'd rather do something like TypemodelPackage.Literals.ICASE_SENSITIVE_NAMED_ELEMENT.isSupertypeOf(eClass), which should be nicer and probably faster

Reply:
This cannot be done, then a cycle in the MANIFEST dependencies appears